index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/x-icon" href="./public/favicon.ico">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <link rel="stylesheet" href="./styles.css">
+    <script src="/script.js" defer></script>
```

Comment:
Script-i fayl@ karox eq body-i mej kapel naev. Iharke ays tarberakovel e hnaravor uxxaki aydqan voch taracvac inchpes body-in@

Reply:
I prefer to keep the content of the body just for markup, and put all the scripts and links in the head tag.
But if you have any suggestions why I should move it to the body tag and what benefits I will get as a result, you can specify them and I will move it.